src/data-workspace/inputs/validators.js

```diff
@@ -87,13 +87,19 @@ export const validatorsByValueType = {
     [VALUE_TYPES.URL]: url,
 }
 
-export const validateByValueTypeWithLimits = (valueType, limits) => {
+export const validateByValueType = (valueType) => {
     const validators = []
 
     if (validatorsByValueType[valueType]) {
         validators.push(validatorsByValueType[valueType])
     }
 
+    return composeValidators(...validators)
+}
+
+export const warningValidateByValueType = (valueType, limits) => {
+    const validators = []
+
     if (
         CAN_HAVE_LIMITS_TYPES.includes(valueType) &&
         Number.isFinite(limits?.min) &&
```

Comment:
Finish refactoring ;) `composeValidators` is kinda janky to use on 0 or 1 validator